tests/test_distance_transform_edt.py

```diff
@@ -0,0 +1,141 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import annotations
+
+import unittest
+
+import numpy as np
+from parameterized import parameterized
+
+from monai.transforms import DistanceTransformEDT, DistanceTransformEDTd
+from tests.utils import HAS_CUPY, assert_allclose, optional_import, skip_if_no_cuda
+
+momorphology, has_cucim = optional_import("cucim.core.operations.morphology")
+ndimage, has_ndimage = optional_import("scipy.ndimage")
+cp, _ = optional_import("cupy")
+
+TEST_CASES = [
+    [
```

Comment:
Maybe we should also have some test cases with C and B > 1. Just repeating the axis might be fine for this.

Reply:
You mean for the shape of the input? I can add those, that's not much work now. I would just add (3,5,5) then

Reply:
Yes with batch and channel dimensions > 1

Reply:
We should also test different datatypes.